### PR TITLE
add metallb to zero cluster

### PR DESCRIPTION
**Pull request by larsks**
install the metallb [1] load balancer on the zero cluster

requires operate-first/apps#405

[1] https://metallb.universe.tf/

### CODE REVIEW

File: envs/moc/zero/cluster-management/metallb.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: metallb
+spec:
+  destination:
+    name: zero
+    namespace: argocd
```
Comment (tumido) [inline envs/moc/zero/cluster-management/metallb.yaml:8]: I think this app doesn't deploy into this namespace. Should be `metallb`

**Review by tumido — DISMISSED** [review]

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Pull request merged**